### PR TITLE
Add ability to set i3-nagbar font

### PR DESCRIPTION
**Pull request by randomguy**
This is a proposal to resolve issue #665.

A section named 'custom' has been added as a valid py3status configuration section. If it contains a key named 'nagbar_font', its value will be used as an argument to the i3-nagbar -f option.

### CODE REVIEW

**Comment by tobes** [comment]
Looks nice and clean.  My only thought is about the section name.  I think I'd prefer `py3status` over `custom`, maybe even leave it in general but that probably causes more work due to i3status also using that section.

**Comment by randomguy** [comment]
You are right, custom was to generic. I adjusted it and added some documentation.

**Comment by randomguy** [comment]
So how is the documentation now? Any more feedback?

File: py3status/parse_config.py [hunk 1]
```diff
@@ -563,6 +563,8 @@ def parse_config(config):
         general_defaults.update(config_info['general'])
     config['general'] = general_defaults
 
+    if 'py3status' in config_info:
+        config['py3status'] = config_info['py3status']
```
Comment (tobes) [inline py3status/parse_config.py]: How about just

`config['py3status'] = config_info.get('py3status', {})`

that way config['py3status'] will always exist and be a dict

File: py3status/core.py [hunk 2]
```diff
@@ -408,7 +409,12 @@ def notify_user(self, msg, level='error', rate_limit=None, module_name=''):
                 cmd = ['notify-send', '-u', DBUS_LEVELS.get(level, 'normal'),
                        '-t', '10000', 'py3status', msg]
             else:
-                cmd = ['i3-nagbar', '-m', msg, '-t', level]
+                config = self.i3status_thread.config
+                nagbar_font = config.get('py3status').get('nagbar_font')
```
Comment (tobes) [inline py3status/core.py]: `nagbar_font = config.get('py3status').get('nagbar_font')`

will fail if `config['py3status']` is `None`. `nagbar_font = config.get('py3status', {}).get('nagbar_font')` would be fine but my other comment fixes this issue

**Comment by randomguy** [comment]
Changed as proposed.

**Comment by tobes** [comment]
@ultrabug I think that adding the new `py3status` block to the config is a good plan.  I can see it having other uses.

**Comment by tobes** [comment]
@ultrabug I think the `pystatus` section would be really helpful for other settings too.  The nagbar font is a bonus.

**Comment by ultrabug** [comment]
Interesting point, I agree that it can be useful, thanks a lot @randomguy !!

**Pull request merged**